Gopkg.toml

```diff
@@ -6,10 +6,10 @@ required = [
 
 [[constraint]]
   name = "github.com/tektoncd/pipeline"
-  revision = "558b37edfffa1d8a33428fb9bd50a8ab425e3e83"
+  revision = "60d76bd5526e815cdbf5639ff3fa5b9eb09ce321"
```

Comment:
why not using version now ? `version = "v0.11.0-rc1"`

Reply:
Fixed 🤗